### PR TITLE
Fix nits from previous PRs

### PR DESCRIPTION
**Pull request by Tbaut**
- Show names on staking overview used to show `VALIDATOR (STASH)` even for known accounts.
- Account address in lowercase for "account or names"
- fix account cards off
- Selection wasn't showing in changeRewardDestination modal.
- avoid the edit or copy icon to get wrapped alone eg (before/after) :
image

image

### CODE REVIEW

**Comment by jacogr** [comment]
Not sure about the first one at all -

image

**Comment by Tbaut** [comment]
Do you mean the second address of your screen shot (5DqAz..) is actually part of your address book or accounts? I doubled checked and it shows as expected for me also on Alex. e.g a known address, my account, and finally an unknown one

image

**Comment by jacogr** [comment]
The first one (this is on Alex) is a validator I added to my addressbook. But now that you mention it, it does not show the value from accounts, i.e. my `First validator` shows up, however `Alice_stash` does not.

File: packages/ui-app/src/Row.tsx [hunk 1]
```diff
@@ -109,14 +109,18 @@ export const styles = `
...
     box-sizing: border-box;
     margin: 0;
     padding: 0;
     overflow: hidden;
     text-overflow: ellipsis;
     text-transform: uppercase;
     white-space: normal;
+
+    .withAddr {
+      text-transform: lowercase;
```
Comment (jacogr) [inline packages/ui-app/src/Row.tsx]: This is not correct - addresses have uppercase letters as well.

Comment (jacogr) [reply packages/ui-app/src/Row.tsx]: For AddressMini, this is what was done - https://github.com/polkadot-js/apps/blob/master/packages/ui-app/src/AddressMini.tsx#L137

Apply the text-transform to `withName` only, so `withAddr` has just the default.

**Comment by Tbaut** [comment]
Thanks, I missed the one with explorer link. Had a look around and found similar problems in council where the name would have been displayed only for address book contact but not your own accounts.

**Comment by jacogr** [comment]
Thank you, wanted to check those all - think we went a bit heavy-handed on applying the type.

**Comment by Tbaut** [comment]
I'll stop here for this PR to keep it small

File: packages/ui-app/src/CopyButton.tsx [hunk 2]
```diff
@@ -85,4 +87,8 @@ export default styled(CopyButton)`
   button.ui.mini.icon.primary.button.iconButton {
     cursor: copy;
   }
+
+  .copySpan {
```
Comment (jacogr) [inline packages/ui-app/src/CopyButton.tsx]: Does this really make a difference? The span is basically around a single element? (A button with only an icon, which should not be breakable)

File: packages/ui-app/src/Row.tsx [hunk 3]
```diff
@@ -41,6 +41,9 @@ export const styles = `
     z-index: 1;
   }
 
+  .editSpan {
+    white-space: nowrap;  }
```
Comment (jacogr) [inline packages/ui-app/src/Row.tsx]: (nit) misplaced `}` at the end.

File: packages/ui-app/src/CopyButton.tsx (hunk 2 above)
Comment (Tbaut) [reply packages/ui-app/src/CopyButton.tsx]: This is black magic.. Found the trick *on the internet* and it does work on both chrome and FF :man_shrugging:

**Pull request merged**